modules/node-types.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='node-types-{context}']
+= Node types in {product-title}
+
+{product-title} uses bootstrap, master, and worker nodes. The bootstrap node
```

Comment:
Haven’t settled on naming yet for worker nodes.  I would however say that we don’t “use ... nodes”, we assign nodes roles within the cluster that define their security boundaries and have specific roles.

It’s unclesr to me that this is the best way to communicate this though - bootstrap nodes are a detail of installation, while master and worker/compute nodes are what an admin deals with every day.  I wouldn’t give them the same level of importance.

This page would be better as a discussion of the roles nodes play (use role instead of type, role is the official term).  Connecting it to the ways people can change nodes (masters today cannot be changed, machines sets mean adding new nodes roles is easy), how security boundaries work, what tools are used to subdivide and group nodes is better.

Reply:
@smarterclayton ^^